src/components/LayerUI.tsx

```diff
@@ -126,6 +126,10 @@ const LayerUI = ({
     jotaiScope,
   );
 
+  useEffect(() => {
+    document.title = `${appState?.name} | Excalidraw | Hand-drawn look & feel • Collaborative • Secure`;
```

Comment:
```suggestion
    document.title = `${appState?.name} | ${APP_NAME}`;
```

Reply:
Sure, will do it this weekend.

btw sorry for the late reply.